_posts/2019-12-19-upgrading-to-catalina.md

```diff
@@ -0,0 +1,15 @@
+# Upgrading to Catalina
+
+I have waited a few months before upgrading to upgrade to Catalina, as I do every single year before upgrading to any new version of the OS.
+
+For now I'll just upgrade the mac I use at work since I want to use Swift UI live preview's. If it was not for that I doubt I would upgrade for a while longer. This year's release doesn't really add any worthwhile user facing feature.
+
+We are not using them at work yet and I doubt that will happen for a few years since we still need to support old OS versions, but it is nice to have them ready and being able to test them out and play around with them.
```

Comment:
Using what?

Reply:
I think that's fine as well since it referers to the previous sentence...?